app/models/concerns/taxonomix.rb

```diff
@@ -224,6 +230,16 @@ def ensure_taxonomies_not_escalated
     end
   end
 
+  def add_taxonomies_having_select_all_enabled
+    [Organization, Location].each do |taxonomy_class|
```

Comment:
Please use `Taxonomy.enabled_taxonomies` if possible, some users might have enabled only a subset of taxonomies.